Gems/AWSCore/Code/Tests/ScriptCanvas/AWSScriptBehaviorS3Test.cpp

```diff
@@ -145,17 +145,18 @@ TEST_F(AWSScriptBehaviorS3Test, GetObjectRaw_CallWithOutfileDirectoryNoExist_Inv
     AWSScriptBehaviorS3::GetObjectRaw("dummyBucket", "dummyObject", "dummyRegion", dummyDirectory);
 }
 
+#if defined(AZ_DEBUG_BUILD) || defined(AZ_PROFILE_BUILD)
+// The preparation step for this test case does not work in release mode
```

Comment:
Curious: Could you provide more details on this (i.e. the preparation step you mean here)? Is this a test specific issue or a bug for release build?

Reply:
It is a test specific issue, try to create a read-only file for this test case in setup step, as we want to test write into a read-only file. But seems like in release version, the read-only file is failed to get created so just disable this test case for release build. (as unit test cover the functionality check in debug and profile build)